src/Microsoft.Extensions.FileProviders.Abstractions/NullFileProvider/NullFileInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Extensions.FileProviders.Abstractions
+{
+    public class NullFileInfo : IFileInfo
+    {
+        public NullFileInfo(string name)
+        {
+            Name = name;
+        }
+
+        public bool Exists => false;
+
+        public bool IsDirectory => false;
+
+        public DateTimeOffset LastModified => DateTimeOffset.MinValue;
+
+        public long Length => -1;
+
+        public string Name { get; }
+
+        public string PhysicalPath => null;
+
+        public Stream CreateReadStream()
+        {
+            throw new FileNotFoundException(string.Format($"{nameof(NullFileProvider)} does not support reading files.", Name));
```

Comment:
Why are you using string interpolation AND `string.Format` here? Also, you pass "Name" as the second argument to `string.Format`, but there is no `{0}`.